whois-client/src/main/java/net/ripe/db/whois/api/rest/domain/WhoisObject.java

```diff
@@ -24,6 +27,7 @@
         "resourceHolder",
         "abuseContact",
         "managed",
+        "objectInfoMessages",
```

Comment:
We already have `net.ripe.db.whois.api.rest.domain.ErrorMessages` for top-level error messages (and we can't change that without changing the API). Can you name this field `InfoMessages` for this situation to be consistent rather than `ObjectInfoMessages` ? And use a `List<InfoMessage>` rather than String so we can extend it later if we need to.

Reply:
As discussed create an abstract `Messages` class and extend with `ErrorMessages` and `InfoMessages`

Reply:
Done